client/layout/guided-tours/style.scss

```diff
@@ -13,7 +13,7 @@
 	animation-name: guided-tours__step-fadein;
 	animation-timing-function: ease-in-out;
 
-	.guided-tours__step-text {
+	p {
```

Comment:
This isn't totally in line with our CSS guidelines, but I think in this case it could be justified — since the goal here is to simplify an API.